scripts/mysql/schema-0.5.0-mysql.sql

```diff
@@ -135,10 +135,7 @@ CREATE TABLE IF NOT EXISTS `role_meta` (
     `role_name` VARCHAR(128) NOT NULL COMMENT 'role name',
     `metalake_id` BIGINT(20) UNSIGNED NOT NULL COMMENT 'metalake id',
     `properties` MEDIUMTEXT DEFAULT NULL COMMENT 'schema properties',
-    `securable_object_full_name` VARCHAR(256) NOT NULL COMMENT 'securable object full name',
```

Comment:
If you need to modify the structure of the MySQL table, in the long run, I think it is best to provide some additional `alter table` statements, such as creating an `upgrade-0.5.0-to-0.5.1-mysql.sql` file to help users of lower versions can be upgraded smoothly. You can create a new version of the schema sql file, such as `schema-0.5.1-mysql.sql`, so that users can directly use the new schema when accessing version 0.5.1.

Reply:
This part isn't available. Maybe we don't need to provide it now. Once we released this feature, we should provide alter scripts.

Reply:
The layout is shipped with 0.5.0, even it is not available from feature level, we still need to provide a script to support users to upgrade their storage layout. You can refer to #3506 for the solution.